fontforge/views.h

```diff
@@ -1330,16 +1330,6 @@ extern void _CVMenuNameContour(CharView *cv);
 extern void Prefs_LoadDefaultPreferences( void );
 
 
-extern CharView* CharViewFindActive();
-extern FontViewBase* FontViewFindActive();
-extern FontViewBase* FontViewFind( int (*testFunc)( FontViewBase*, void* ), void* udata );
-
-extern int FontViewFind_byXUID(      FontViewBase* fv, void* udata );
-extern int FontViewFind_byXUIDConnected( FontViewBase* fv, void* udata );
-extern int FontViewFind_byCollabPtr(  FontViewBase* fv, void* udata );
-extern int FontViewFind_bySplineFont( FontViewBase* fv, void* udata );
-extern int FontViewFind_byCollabBasePort( FontViewBase* fv, void* udata );
-
```

Comment:
Are you sure that these functions (other than the collab one) are worthless (unlikely to be used)?

Reply:
Yes, they were all introduced with collab support, but this was missed when removing it.